bin/schematic.py

```diff
@@ -39,12 +40,13 @@ def parse_schematic(path, open_func):
 
 def build_slice(slice, block_defs):
     selected = 0
-    turn_dir = turtle.turnRight
+    turn_dir = nav.TURNS.RIGHT
+    turn = navigator.turn
```

Comment:
No need to assign like this. Just call `navigator.turn` later

Reply:
Also is there any reason navigator needs to handle basic left and right turns?

Seems like turtle.turnLeft and turtle.turnRight do the exact same thing